esm_runscripts/filelists.py

```diff
@@ -136,11 +136,15 @@ def complete_targets(config):
                         # check if the file_source has the correct type. For
                         # unresolved variables they may still be a 'dict'
                         if not isinstance(file_source, (str, os.PathLike)):
+                            # model, scenario and version are omitted to make
+                            # error message less verbose
                             scenario = config[model].get('scenario', 'UNDEFINED')
                             version = config[model].get('version', 'UNDEFINED')
+
                             error_type = "Missing Scenario Configuration"
```

Comment:
The error type needs to be renamed if we remove the Scenario details above. Suggestion: Missing Configuration i.e. remove Scenario from the text.